Interni/StudioDiFattibilità/components/fattibilità-c1.tex

```diff
@@ -0,0 +1,72 @@
+\section{Capitolato C1}
+
+\subsection{Informazioni generali}
+\begin{itemize}
+\item \textbf{Nome:} BlockCOVID
+\item \textbf{Proponente:} Imola Informatica
+\item \textbf{Committenti:} \committenti{}
+\end{itemize}
+
+\subsection{Descrizione del capitolato}
+Il capitolato vuole realizzare un sistema che permetta di tutelare la salute e la sicurezza dei lavoratori dal rischio contagio e garantire la salubrità dell'ambiente di lavoro. E' chiesto dunque di realizzare un apparato che soddisfi:
+\begin{itemize}
+\item il tracciamento immutabile e certificato delle presenze in tempo reale nelle postazioni di lavoro di un laboratorio informatico, contrassegnate tramite dei tag RFID;
+\item il tracciamento immutabile e certificato della pulizia delle postazioni.
+\end{itemize}
+\subsection{Finalità del progetto}
+Il prodotto e' composto dalle seguenti parti:
+\begin{itemize}
+\item Il server centrale, gestisce le varie stanze e postazioni, deve poter:
```

Comment:
"il server centrale, **che** gestisce le varie stanze e postazioni [...]", senza il che risulta un po' sconnessa la frase tra le due virgole

Reply:
**deve poter** va in contrasto logico e grammaticale con gli ultimi due punti "deve poter: deve essere possibile"